yt/yt/server/lib/hydra/file_changelog_index.cpp

```diff
@@ -36,6 +36,7 @@ TFileChangelogIndex::TFileChangelogIndex(
     , FileName_(std::move(fileName))
     , Config_(std::move(config))
     , WorkloadCategory_(workloadCategory)
+    , WorkloadDescriptor_(workloadCategory, 0, {}, {}, {}, "TFileChangelogIndex", {})
```

Comment:
you must explicitly create a separate constructor for this type of call. It is unclear what the three empty parentheses mean.

Reply:
builder (with "With") looks better as it seems to me